### DISCUSSION TITLE
Entering the Golden Age of U.S. Women's Soccer

### DISCUSSION THREAD

**Discussion by timothymodel** [has2k1/plotnine#861] [2024 Plotnine Contest]
### Authors

Timothy A. Model

### Links

https://github.com/timothymodel/nwsl_golden_age/tree/main

### Full description

**Full Description**: https://github.com/timothymodel/nwsl_golden_age/blob/main/README.md

![golden_age_of_womens_soccer](https://github.com/user-attachments/assets/198aeb1a-b667-4d12-a33c-6fcef3ef9e55)

With journalists and enthusiasts declaring that "Soccer is taking over America" [(Ikram 2023)](https://fortune.com/2023/09/29/soccer-taking-over-america-apple-win-sports-streaming-wars-mls-deal-aden-ikram/) and with Apple investing in live match streaming in the United States, I wanted to add some color and complexity to the trends we are seeing in soccer's growing popularity.

This growth is not limited to men's soccer, typically represented by Major League Soccer (MLS). Women's soccer, represented by the National Women's Soccer League (NWSL), is also experiencing a significant surge of investment and interest across the country [(Adgate 2023)](https://www.forbes.com/sites/bradadgate/2023/11/14/the-nwsl-had-a-record-breaking-year-with-more-growth-is-expected/).

My goal was to explore the differential trends in attendance between MLS and NWSL teams.

The plot I produced highlights three key points:

1. Men's soccer (MLS) is more popular than women's soccer (NWSL).
2. In most cities, men's soccer (MLS) is still growing.
3. Women's soccer (NWSL) match attendance is lower than men's match attendance (NWSL), BUT, in many cities NWSL attendance is rapidly catching up to MLS attendance.

### Code repository

https://github.com/timothymodel/nwsl_golden_age/tree/main

**Comment by has2k1** [comment]
![golden_age_of_womens_soccer](https://github.com/user-attachments/assets/d12f8103-51ce-41e0-b665-52131a4885f7)

**Comment by EconomiCurtis** [comment]
Ahoy @timothymodel , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@timothymodel, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
